g2o/types/slam3d/edge_pointxyz.h

```diff
@@ -74,7 +74,7 @@ namespace g2o {
       }
 
 
-      virtual number_t initialEstimatePossible(const OptimizableGraph::VertexSet& , OptimizableGraph::Vertex* ) { return 0.;}
```

Comment:
[I'm not going to mention every one, but I think you have a couple more like this]